cql3/statements/alter_keyspace_statement.cc

```diff
@@ -276,7 +276,7 @@ cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_proce
                     locator::replication_strategy_params(ks_md_update->strategy_options(), ks_md_update->initial_tablets()));
 
```

Comment:
> * the load balancer is adjusted to reject any cross-rack migrations

"reject" seems the wrong word here. The load balancer initiates migrations, so it wouldn't need to reject its own migrations.